[WBH] Documentation dedicated ip (#2550)
scaleway/docs-content: managed-services/webhosting/how-to/order-webhosting.mdx

  - You have [registered a domain name](/network/domains-and-dns/how-to/register-internal-domain/) at Scaleway or another registrar
</Message>

1. Click **Web Hosting** within the **Managed Services** section of the side menu. You will be directed to the Web Hosting splash page.
   <Message type="note">

>       If you have existing Web Hosting plans in your account, you will encounter a list of these. Click the <Icon name="plus" /> button to create additional Web Hosting plans.

Reviewer: ```suggestion
      The list of your existing Web Hosting plans will display if you already have any.  Click the <Icon name="plus" /> button to create additional Web Hosting plans.
```

   </Message>
2. Select **Order Web Hosting**, and the Web Hosting creation wizard appears.
3. Choose a **region** and a **control panel** for your Web Hosting. Then click **Configure Web Hosting**.
4. Follow these steps in the wizard:
    - Enter a **domain name** to associate with your Web Hosting plan.
      <Message type="important">

>         The domain must be pre-registered, and you must be the registrant. If you do not have a domain yet, please [register one](/network/domains-and-dns/how-to/register-internal-domain/) before proceeding.

Reviewer: ```suggestion
        The domain must be pre-registered, and you must be the registrant. If you do not have a domain yet, you need to [register one](/network/domains-and-dns/how-to/register-internal-domain/) before proceeding.
```

  - You have [registered a domain name](/network/domains-and-dns/how-to/register-internal-domain/) at Scaleway or another registrar
</Message>

> 1. Click **Web Hosting** within the **Managed Services** section of the side menu. You will be directed to the Web Hosting splash page.

Reviewer 2: ```suggestion
1. Click **Web Hosting** within the **Managed Services** section of the side menu. You are directed to the Web Hosting splash page.
```

   <Message type="note">
      If you have existing Web Hosting plans in your account, you will encounter a list of these. Click the <Icon name="plus" /> button to create additional Web Hosting plans.
   </Message>
2. Select **Order Web Hosting**, and the Web Hosting creation wizard appears.

> 3. Choose a **region** and a **control panel** for your Web Hosting. Then click **Configure Web Hosting**.

Reviewer 2: ```suggestion
3. Choose a **region** and a **control panel** for your Web Hosting plan. Then click **Configure Web Hosting**.
```
